src/backend/app.js

```diff
@@ -21,10 +22,86 @@ app.use(cors());
 
 router.use("/meals", mealsRouter);
 
+//Respond with all meals in the future (relative to the when datetime) - /future-meals
+
+app.get("/future-meals", async (req, res) => {
+  try {
+    const today = new Date();
+    const futureMeals = await knex("meal").where("event_date", ">", today);
+    if (futureMeals.length === 0) {
+      res.status(204).json(futureMeals);
+    }
+    res.status(200).json(futureMeals);
+  } catch (error) {
+    console.error(error);
+    res.status(500).send("Server Error");
+  }
+});
+
+//Respond with all meals in the past (relative to the when datetime) - /past-meals
+
+app.get("/past-meals", async (req, res) => {
+  try {
+    const today = new Date();
+    const pastMeals = await knex("meal").where("event_date", "<", today);
+    if (pastMeals.length === 0) {
+      res.status(204).json(pastMeals);
```

Comment:
it makes sense to send a message with a response like 
"meal not found" or something like this